src/components/tooltip/Marker.js

```diff
@@ -714,6 +717,43 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
       implementationFilter,
       zeroAt
     );
+
+    // FileIO markers can belong to different threads.
+    let threadDetail = null;
+    const data = marker.data;
+    if (data && data.type === 'FileIO' && data.threadId !== undefined) {
```

Comment:
I'm not convinced we should keep `data.type === 'FileIO'`. What do you think about removing it at all? You can still mention FileIO markers in a comment, but more as an example.

Reply:
Yeah, that's right removed it.